e2e/tests/localAuthoritySubmitsApplication_@slow_test.js

```diff
@@ -231,25 +232,27 @@ Scenario('local authority enters respondents @create-case-with-mandatory-section
   I.seeInTab(['Respondents 1', 'Party', 'Last name'], respondents[0].lastName);
   I.seeInTab(['Respondents 1', 'Party', 'Date of birth'], '1 Jan 1980');
   I.seeInTab(['Respondents 1', 'Party', 'Gender'], respondents[0].gender);
-  I.seeInTab(['Respondents 1', 'Party', 'Place of birth'], respondents[0].placeOfBirth);
   I.seeInTab(['Respondents 1', 'Current address', 'Building and Street'], respondents[0].address.buildingAndStreet.lineOne);
   I.seeInTab(['Respondents 1', 'Current address', 'Address Line 2'], respondents[0].address.buildingAndStreet.lineTwo);
   I.seeInTab(['Respondents 1', 'Current address', 'Address Line 3'], respondents[0].address.buildingAndStreet.lineThree);
   I.seeInTab(['Respondents 1', 'Current address', 'Town or City'], respondents[0].address.town);
   I.seeInTab(['Respondents 1', 'Current address', 'Postcode/Zipcode'], respondents[0].address.postcode);
   I.seeInTab(['Respondents 1', 'Current address', 'Country'], respondents[0].address.country);
   I.seeInTab(['Respondents 1', 'Telephone', 'Telephone number'], respondents[0].telephone);
-  I.seeInTab(['Respondents 1', 'What is the respondent\'s relationship to the child or children in this case?'], 'mock reason');
+  I.seeInTab(['Respondents 1', 'What is the respondent\'s relationship to the child or children in this case?'], respondents[0].relationshipToChild);
   I.seeInTab(['Respondents 1', 'Do you need contact details hidden from other parties?'], 'No');
   I.seeInTab(['Respondents 1', 'Do you believe this person will have problems with litigation capacity (understanding what\'s happening in the case)?'], 'Yes');
   I.seeInTab(['Respondents 1', 'Give details, including assessment outcomes and referrals to health services'], 'mock reason');
+  I.seeInTab(['Respondents 1', 'Representative', 'Representative\'s first name'], respondents[0].solicitor.firstName);
+  I.seeInTab(['Respondents 1', 'Representative', 'Representative\'s last name'], respondents[0].solicitor.lastName);
+  I.seeInTab(['Respondents 1', 'Representative', 'Email address'], respondents[0].solicitor.email);
+  //I.seeInTab(['Respondents 1', 'Representative', 'Organisation'], 'Swansea City Council');
 
```

Comment:
why is this one commented out?

Reply:
not sure 🤔 fixing

Reply:
not trivial to fix as complex type for organisation is special and has different xpath in tab compared to normal field, maybe can be fixed in future tab story, as it is not a blocker here

Reply:
happy with this